app/instance.py

```diff
@@ -133,7 +133,7 @@ def terminate_resource(self, dryrun: bool) -> bool:
             return True
         except Exception as e:
             print(f'Failure when calling terminate_instances: {str(e)}')
-            return False
+            return e
```

Comment:
method return indicates `bool` whereas `e` is `Exception`

Reply:
Made a change to remove the return type from the method, and returned the exception as a `string` rather than an `Exception'.